javatests/build/bazel/tests/integration/BUILD

```diff
@@ -40,8 +43,5 @@ bazel_java_integration_test(
         "@test_archive",
         "@test_archive2",
     ],
-    # Ensure that there could be no network fetch in sandboxing.  This tag is a Bazel
-    # built-in, it is important to keep it.
-    tags = ["block-network"],
```

Comment:
Why did you need to remove this tag? Does this make it not hermetic?

Reply:
Because with newer version of Bazel, `WorkspaceDriverIntegrationTest` will fail with "unknown host" when it tries to access "mirror.bazel.build", I'm not sure how to deal with this without removing this "block-network" tag. @ittaiz Can you give some instructions?